cmd/podman/shared/create.go

```diff
@@ -41,6 +41,9 @@ func CreateContainer(ctx context.Context, c *GenericCLIResults, runtime *libpod.
 		span, _ := opentracing.StartSpanFromContext(ctx, "createContainer")
 		defer span.Finish()
 	}
+	if c.Bool("rm") && c.String("restart") != "" && c.String("restart") != "no" {
+		return nil, nil, errors.Errorf("the --rm flag conflicts with --restart")
```

Comment:
nit, may be personal, but we list `--rm' as an option on the man page and I think we should refer to them as an option rather than a flag on external messaging.